genny-project/layouts
# Feature intm-748

# Issue https://outcomelife.atlassian.net/browse/INTM-748

# Description:
When not using custom login layouts, the splash screen is redundant. This replaces the splash screen layout with a redirect to the keycloak login page.

# How To Test
1) Go to `localhost:3000` or `alyson.genny.life` in Chrome incognito. Ensure there is not `/home` or any other additions to the route.
2) Page should navigate to the keycloak login page.
3) Logout and go to `localhost:3000/home` or `alyson.genny.life/home`.
4) Page should navigate to the keycloak login page.
